smk/assembly.smk

```diff
@@ -242,6 +243,18 @@ rule correct_spadeshammer:
 # Get a sorted list of runs for a sample
 
 def get_runs_for_sample(wildcards):
+    # If corrected runs are already present, take it from there
```

Comment:
I don't think this is necessary to add. Runs are not renamed in the correction step, so it would yield the same run strings, as the one for the "uncorrected" below.

Reply:
True. I added this to catch our use-case in the workshop - when we remove `4-hostfree` to save space. I could make it more concise in a loop since the `runs=[...]` step is exactly the same.

Reply:
For that special case, could it be done in a similar way as in gene_abundance.smk instead?

Reply:
I pushed a new update for this. Think it's simplified now.